prepare/acroname.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
```

Comment:
Would it be possible to store the file in tmp? prepare is for env preparation scripts, we do copy acroname.py to /tmp anyway.